docs/docs/migration_notes.md

```diff
@@ -44,10 +48,6 @@ Further changes are planned, so that:
 
 ## 0.66
 
-### L2 Gas limit of 12M enforced for public portion of TX
-
-This limit was previously enforced per-enqueued-public-call. The protocol now enforces a stricter limit that the entire public portion of a transaction consumes at most 12,000,000 L2 gas. 
-
```

Comment:
This was incorrectly placed in the `0.66` section by a previous PR because i thought it would be merged before 0.66....